common/views/components/BasePage/EventPage.js

```diff
@@ -91,6 +99,10 @@ function topDate(event) {
   );
 };
 
+function showTicketSalesStart(dateTime) {
+  return dateTime && !isTimePast(dateTime);
+}
```

Comment:
Is this abstraction necessary?

Reply:
perhaps not

Reply:
> although it does make the template slightly simpler to reason about

Not sure, as the method name `showTicketSalesStart` doesn't really give any extra meaning?